src/inference/io_adapter.py

```diff
@@ -1,6 +1,8 @@
 import abc
 import os
 
+import json
+import pathlib
```

Comment:
```suggestion
from pathlib import Path
```
иначе очень много всего заимпортится